cross-lib/run_enki_on_all.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+while [ -n "$1" ]; do
+  case "$1" in
+    --command) 
+      shift; arg_command=$1
+    ;;
+    *)
+      echo "Invalid argument $1"
+      exit 1
+    ;;
+  esac
+  shift
+done
+
+[[ $arg_command ]] || ( echo "ERROR: A command was not provided. Typical examples are 'all-git-pdf' or 'all-git-web' or 'all-git-epub'" && exit 1 )
+
+all_books="cross-lib/book_data/AUTO_books.txt"
+test -f $all_books || ( echo "ERROR: Book list not found at ${all_books}" && exit 1 )
```

Comment:
It's book-data in the ruby script. Initially, I ran the ruby script to update books, so this became a problem.
```bash
all_books="cross-lib/book-data/AUTO_books.txt"
```